doc/developers/tips.rst

```diff
@@ -103,6 +103,11 @@ replies <https://github.com/settings/replies/>`_ for reviewing:
 ..
     Note that putting this content on a single line in a literal is the easiest way to make it copyable and wrapped on screen.
 
+Issue: Usage questions
+    ::
+
+        You're asking an usage question. The issue tracker is mainly for bugs and new features. For usage questions, it is recommended to try other places such as [Stack Overflow](https://stackoverflow.com/questions/tagged/scikit-learn) and [the Mailing List](https://mail.python.org/mailman/listinfo/scikit-learn), or turn to people around you for help.
```

Comment:
Even it I couldn't find a good reference, according to [this answer](https://answers.yahoo.com/question/index?qid=20110321172251AA8ia1t&guccounter=1) "a usage" is more common than "an usage" 


I would remove "other places such as" and "turn to people around you for help". The latter is not always applicable. Also say if you have a new user in the same office, you wouldn't want him to ask you questions for each issue he runs into before looking on StackOveflow by him/herself..

Reply:
> Even it I couldn't find a good reference, according to this answer "a usage" is more common than "an usage"

I think these are just mistakes from non-native English speakers like me :) updated